doc/ida/guide/source/Usage/index.rst

```diff
@@ -942,53 +942,6 @@ Main solver optional input functions
    +--------------------------------------------------------------------+---------------------------------+----------------+
 
 
-.. c:function:: int IDASetErrFile(void * ida_mem, FILE * errfp)
```

Comment:
Looks like there are still some instances of `IDASetErrFile` and `IDASetErrHandlerFn` elsewhere in this file. Also need to remove `IDAErrHandlerFn` documentation.